make/autoconf/lib-clang.m4

```diff
@@ -102,14 +102,14 @@ AC_DEFUN_ONCE([LIB_SETUP_LIBCLANG],
 
     dnl Only for Windows platform now, as we don't need bin yet for other platform
     if test "x$OPENJDK_TARGET_OS" = xwindows; then
-        BASIC_FIXUP_PATH(CLANG_BIN_PATH)
+        UTIL_FIXUP_PATH(CLANG_BIN_PATH)
     else
         CLANG_BIN_PATH=""
     fi
 
-    BASIC_FIXUP_PATH(CLANG_INCLUDE_PATH)
-    BASIC_FIXUP_PATH(CLANG_LIB_PATH)
-    BASIC_FIXUP_PATH(CLANG_INCLUDE_AUX_PATH)
+    UTIL_FIXUP_PATH(CLANG_INCLUDE_PATH)
+    UTIL_FIXUP_PATH(CLANG_LIB_PATH)
+    UTIL_FIXUP_PATH(CLANG_INCLUDE_AUX_PATH)
 
     if test "x$CLANG_INCLUDE_PATH" != "x"; then
         LIBCLANG_CPPFLAGS="-I$CLANG_INCLUDE_PATH"
```

Comment:
These changes are necessary, as the build seems to be broken after the latest merge from jdk/jdk - these are orthogonal to the fix tackled in this issue